posts/how-to-choose-between-react-angular-or-vue.md

```diff
@@ -0,0 +1,160 @@
+
+
+## Other Things To Consider
+
+As you have seen earlier, each framework or library has it’s pros and cons. You can look at it as, React being the most light weight library and Angular being a more heavy or larger framework. And Vue is somewhere in the middle, not too light, and not too heavy. Vue can also appear to have the lightest learning curve being that it doesn’t use the jsx syntax, and has more shorthand syntax, meaning less code to write. However, easy of use is definitely subjective depending on the individual’s experience and background. Then, there are the less subjective differences such as the amount of support or size of community, out of the box or packaged features, syntax used to achieve the same output within the code. 
+
+When considering support or the size of community; determine if your developers weigh heavily on support and community. How often or frequent do you or your developers lean on help from the community to solve problems. Is using support a huge part of your production process? React and Angular have larger communities than Vue.
+
+Also, are your developers more senior or mid to junior level? Which framework or library does your developers currently have the most knowledge with? Will the complexity of learning the framework or library make a significant impact on the development time or robustness of the application? Moreover, do you have the funds to outsource the development of your application to experienced developers? If the complexity of the framework a major factor, Vue maybe a good choice.
+
+
+## Wrap Up
+
+Is one framework or library better than the other? I would answer 'no' because all these tools have a purpose in the web application development space, depending on your application needs. Remember to consider your application needs most importantly. Then consider all the other secondary factors about the pros and cons of each framework or library, such as the learning curve, syntax, and support. Also, evaluate your developer's experience and level of expertise. It may be more beneficial to work with the framework or library that your developers are most experienced with. This can save you time and labor, by reducing the amount of learning, researching, trial & error, as well as, the potential amount of mistakes or bugs in the code. However, it all boils down to using the right tool for the job. Learn more about the latest features of [React](https://reactjs.org), [Angular.js](https://angular.io), and [Vue.js](https://vuejs.org), from their websites and discover which one will work best for your situation.
```

Comment:
```suggestion
Is one framework or library better than the other? I would answer "no," mainly because all of these tools have a purpose in web application development, all rooting primarily to _your_ application needs. The consider all the other secondary factors about the pros and cons of each framework or library, such as the learning curve, syntax, and support. Evaluate your developers' experience and level of expertise. It may be more beneficial to work with the framework or library that your developers are most experienced with. This can save you time and labor by reducing the amount of learning, researching, trial and error, etc., as well as the potential amount of mistakes or bugs in code. It all boils down to using the right tool for the job. Learn more about the latest features of [ReactJS](https://reactjs.org), [Angular](https://angular.io), and [Vue.js](https://vuejs.org) from their websites and discover which one can work best for your situation.
```

Reply:
The third sentence, first word of the paragraph, you have: The consider all the other secondary factors about the pros and cons of each framework or library, such as the learning curve, syntax, and support. I originally had "Then consider", is that a typo?

Reply:
Yes, my bad!  Should be "Then consider --".  I will update and then you can push.

Reply:
Cool, Thanks